nilearn/decomposition/multi_pca.py

```diff
@@ -147,12 +81,16 @@ class MultiPCA(BaseEstimator, TransformerMixin, CacheMixin):
         The number of CPUs to use to do the computation. -1 means
         'all CPUs', -2 'all CPUs but one', and so on.
 
+    max_nbytes: int,
+        Size (in bytes) above which the intermediary unmasked data will be
+        stored as a tempory memory map
+
     verbose: integer, optional
-        Indicate the level of verbosity. By default, nothing is printed
+        Indicate the level of verbosity. By default, nothing is printed.
 
     Attributes
     ----------
-    `masker_`: instance of MultiNiftiMasker
+    `_pca_masker_`: instance of MultiNiftiMasker
```

Comment:
We exposer the masker because it ca be used by the user to `inverse_transform` components. Why do make it private?